k9mail/src/test/java/com/fsck/k9/controller/MessagingControllerTest.java

```diff
@@ -693,7 +693,7 @@ public void synchronizeMailboxSynchronous_withAccountPolicySetToExpungeOnPoll_sh
 
         controller.synchronizeMailboxSynchronous(account, FOLDER_NAME, listener, null);
 
-        verify(remoteFolder).expunge();
+        verify(remoteFolder).expunge(anySet());
```

Comment:
`anySet()` is pretty weak. Can't we verify we pass in the right IDs?

Reply:
done